src/App.css

```diff
@@ -20,92 +20,196 @@ h2,
+  padding: 0.25em 0.75em;
+  background-color: #e0d4f6;
+  box-shadow: 0 3px 5px rgba(0, 0, 0, 0.18);
+  font-family: inherit;
+  font-size: 1.5rem;
+  line-height: 1.1;
+  text-align: center;
+  cursor: pointer;
+  transition: 220ms all ease-in-out;
+  color: #675883;
+  font-weight: bold;
+  border-radius: 0.15em;
 }
 
 .button__home:hover,
 .button__home:active {
-    background-color: powderblue;
+  background-color: powderblue;
 }
 
 .button__home:focus {
-    outline-style: solid;
-    outline-color: transparent;
+  outline-style: solid;
+  outline-color: transparent;
 }
 
 .form-container {
-    display: grid;
-    grid-template-rows: auto auto auto auto;
-    grid-template-columns: auto;
-    justify-content: center;
-    grid-row-gap: 10px;
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  grid-template-columns: auto;
+  justify-content: center;
+  grid-row-gap: 10px;
 }
 
 .add-button {
-    margin-top: 30px;
-    background-color: #4caf50; /* Green */
-    border: none;
-    color: white;
-    padding: 15px 32px;
-    text-align: center;
-    text-decoration: none;
-    display: inline-block;
-    font-size: 16px;
+  text-rendering: optimizeSpeed;
+  line-height: 1.5;
+  font-size: 1.33rem;
+  box-sizing: border-box;
+  display: inline-block;
+  padding: 0.5rem 2rem;
+  background: #fe9476;
+  color: #263147;
+  border: 2px solid;
+  text-transform: uppercase;
+  text-decoration: none;
+  text-align: center;
+  font-family: 'Red Hat Display', 'Segoe UI', Roboto, 'Helvetica Neue', Arial,
+    sans-serif;
+  font-weight: 900;
+  margin-top: 1.77rem;
+  border-radius: 0.25rem;
+}
+
+.form {
+  display: grid;
+  place-content: center;
+  color: #404040;
+  font-size: 1rem;
+  line-height: 1.375rem;
+  text-align: left;
+}
+
+.form-row {
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeSpeed;
+  margin: 0;
+  padding: 0;
+  color: #404040;
+  list-style: none;
+  direction: ltr;
+  font-size: 1rem;
+  line-height: 1.375rem;
+  text-align: left;
+  box-sizing: inherit;
+}
+
+.form-row + .form-row {
+  margin-top: 1.5rem;
+}
+
+.form-group label {
+  font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier;
+}
+
+.form-field {
+  display: block;
+  width: 100%;
+  max-width: 60ch;
+  margin-top: 0.125rem;
+  padding: 0.85rem 1rem;
+  border: 1px solid;
+  background: none;
+  font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier,
+    monospace;
+  font-size: 1rem;
+  line-height: 1;
+}
+
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+input:-webkit-autofill:active {
+  -webkit-transition: 'color 9999s ease-out, background-color 9999s ease-out';
+  -webkit-transition-delay: 9999s;
 }
 
-.lists {
-    display: grid;
-    justify-content: center;
-    grid-row-gap: 10px;
+.field-error {
+  display: block;
+  padding: 0.25rem;
+  border: 0 solid;
+  font-family: inherit;
+  font-weight: 400;
+  font-style: italic;
+  color: inherit;
```

Comment:
You can skip inheriting color, it's a property that's inherited by default. Inheritance in CSS is very interesting https://developer.mozilla.org/en-US/docs/Web/CSS/inheritance

Reply:
We removed it :slightly_smiling_face: